src/components/Link/styles.module.scss

```diff
@@ -0,0 +1,11 @@
+.disabled {
+  cursor: not-allowed;
+  opacity: 0.5;
+}
+
+.disabled > a {
+  color: currentColor;
+  display: inline-block;  /* For IE11/ MS Edge bug */
```

Comment:
qué bug? link a stack overflow?

Reply:
https://caniuse.com/#feat=pointer-events
Este texto me base en esto: https://css-tricks.com/how-to-disable-links/
```
According to caniuse, this is not supported for Opera Mini and IE<11. IE11 and Edge actually don't support pointer-events unless display is set to block or inline-block. Also, setting pointer-events to none overwrites our nice not-allowed cursor, so now mouse users will not get that additional visual indication that the link is disabled.
```

Reply:
Dylan Alvarez, [17.06.20 22:30]
el comentario ese del css

Dylan Alvarez, [17.06.20 22:30]
yo la verdad lo sacaria, pero si queres dejar aclarado que esta para ie11, linkearia a la seccion de css-tricks que lo aclara